src/clue.js

```diff
@@ -117,10 +117,46 @@ var Patio = {name: 'Patio'}
 
 
 // Characters Collection
-var charactersArray = [drOrchid, drOrchid, profPlum, missScarlet, mrsPeacock, mrMustard];
+var charactersArray = [mrGreen, drOrchid, profPlum, missScarlet, mrsPeacock, mrMustard];
 
 // Rooms' Collection
-var roomsArray = [rope, knife, candlestick, dumbbell, poison, axe, bat, trophy, pistol];
+var roomsArray = [DinningRoom, Conservatory, Kitchen, Study, Library, BilliardRoom, Lounge, Ballroom, Hall, Spa,  LivingRoom, Observatory, Theater, GuestHouse, Patio];
 
 // Weapons Collection
-var weaponsArray = [DinningRoom, Conservatory, Kitchen, Study, Library, BilliardRoom, Lounge, Ballroom, Hall, Spa,  LivingRoom, Observatory, Theater, GuestHouse, Patio];
+var weaponsArray = [rope, knife, candlestick, dumbbell, poison, axe, bat, trophy, pistol];
+
+
+
+//Random Selector
+
+function randomSelector (cardStack) {
+  return cardStack[Math.round(Math.random()*(cardStack.length-1))];
+}
```

Comment:
Your function is perfect, you are using the method round() but the test is using floor() so the expectation of the result is a bit different.
for the number 4.8,
round would return 5
floor would return 4,
You couldn't know that, so you did really well to continue :)
For later, usually the method floor is more used than the round because the probability of result of each number is totally equal.
Here the code with floor:
function randomSelector (cardStack) {
  return cardStack[Math.floor(Math.random()*(cardStack.length))];
}